get-started/index.qmd

```diff
@@ -0,0 +1,312 @@
+
+- **Scalable, extensible, and production-ready**<br>
+No need to switch frameworks as your application grows.
+You can [incorporate Shiny into an existing report](https://quarto.org/docs/dashboards/), start with a [simple application](/docs/express-in-depth.qmd), and [scale up as needed](/docs/express-to-core).
+If something’s missing, you can even [build a custom component](/docs/custom-component-one-off.qmd).
+
+- **Starter templates**<br>
+Get up and running quickly with a [starter template](/templates/).
+
+- **Open source**<br>
+Inspect, adapt, contribute, or join the [forum](https://forum.posit.co/c/shiny) or [discord](https://discord.com/invite/yMGCamUMnS) community! You can check out the [code](https://github.com/posit-dev/py-shiny/) or make your own [extensions](/docs/custom-component-one-off.qmd).
+
+Ready to dive deeper? [Learn more about what makes Shiny unique](what-is-shiny.qmd).
+
+
+## Gallery and templates
+
+Check out the [Shiny Gallery](/gallery/) for inspiration.
+Or kick start a new project with one of our [starter templates](/templates/).
+
+```{=html}
+<link
+  href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+  rel="stylesheet"
+/>
+
+<style>
+.gallery-container {
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+}
+
+.gallery-card {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  background-color: #f8f9fa;
+}
+
+.gallery-title {
+  width: 100%;
+  text-align: center;
+  font-size: 1rem;
+  font-weight: 500;
+  padding: 8px;
+  min-height: 3.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.gallery-item {
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+  border-radius: 12px;
+  box-shadow: 0 4px 10px rgba(0,0,0,0.2);
+}
+
+.gallery-item img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  display: block;
+  transition: opacity 0.3s ease;
+  border-radius: 12px;
+}
+
+.gallery-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.4);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  opacity: 0;
+  transition: opacity 0.3s ease;
+  border-radius: 12px;
+}
+
+.gallery-item:hover img {
+  opacity: 0.5;
+}
+
+.gallery-item:hover .gallery-overlay {
+  opacity: 1;
+}
+
+.gallery-btn {
+  padding: 6px 16px;
+  font-size: 0.85rem;
+  background-color: rgba(255, 255, 255, 0.15);
+  color: white;
+  border: 2px solid rgba(255, 255, 255, 0.8);
+  text-decoration: none;
+  font-weight: 600;
+  border-radius: 30px;
+  transition: all 0.25s ease;
+  backdrop-filter: blur(4px);
+  -webkit-backdrop-filter: blur(4px);
+  position: relative;
+  z-index: 1;
+}
+
+.gallery-btn:hover {
+  background-color: white;
+  color: black;
+  transform: scale(1.05);
+  box-shadow: 0px 4px 10px rgba(255, 255, 255, 0.3);
+  border-color: white;
+}
```

Comment:
not sure why the bottom part of the button here isn't showing up as white. It's also causing the opposite effect on the actual hover over

![Kapture 2025-04-22 at 10 02 22](https://github.com/user-attachments/assets/f0f4398a-401c-4fe2-984b-3b1ae8f26ec1)

Reply:
This was an interesting idea to explore, thank you! In the end, though, I don't think this improves the experience at the same time that we'd need to spend more time on this to get accessibility and design right. I think we're better off spending our time elsewhere. Can you please restore the previous version?

Reply:
Also the "code" button should take you to the templates page if the app comes from a template

Reply:
So let's change "View code on GitHub" to "View code" and point to templates or github (preference in that order)